sources/api/netdog/src/net_config/v3.rs

```diff
@@ -0,0 +1,129 @@
+use super::v2::NetInterfaceV2;
```

Comment:
This will all drastically change when I rebase on the bonding/vlan changes.  I only added an additional version so I could add unit tests and actually use the new types.